.github/workflows/main.yml

```diff
@@ -363,13 +356,6 @@ jobs:
           distribution: 'temurin'
           java-version: '17'
 
-      # Download all the stuff...
-      - name: Download maven artifacts
-        uses: actions/download-artifact@master
-        with:
-          name: maven
-          path: dist/maven
-
```

Comment:
Hmm, I noticed `Deploy to github package registry` uses artifacts from dist/maven

https://github.com/jMonkeyEngine/jmonkeyengine/blob/44a50de4d7797dc3dd8d46cd1f51308e9a05d627/.github/workflows/main.yml#L368-L373